src/strongbox/ui/jfx.clj

```diff
@@ -484,12 +485,43 @@
 
                "#status-bar"
                {:-fx-font-size ".9em"
-                :-fx-padding "5px"
+                :-fx-padding "0"
+                :-fx-alignment "center-left"
+                ;;:-fx-background-color "pink"
+                }
+
+               "#status-bar-left"
+               {;;:-fx-background-color "orange"
+                :-fx-padding "0 10"
+                :-fx-alignment "center-left"
+                :-fx-pref-width 9999.0
 
                 " > .text"
                 {;; omg, wtf does 'fx-fill' work and not 'fx-text-fill' ???
                  :-fx-fill (colour :table-font-colour)}}
 
+               "#status-bar-right"
+               {:-fx-min-width "130px" ;; long enough to render "warnings (999)"
+                ;;:-fx-background-color "blue"
+                :-fx-padding "5px 12px 5px 0"
+                :-fx-alignment "center-right"}
+
+               "#status-bar-right .button"
+               {;;:-fx-background-color "green"
+                :-fx-padding "4 10"
+                ;; doesn't look right when button is coloured.
+                ;;:-fx-background-radius "4"
+                :-fx-font-size "11px"}
+
+               ".button.with-warning"
+               {:-fx-background-insets "0 0 -1 0,  0,  1,  2"
+                :-fx-background-color (str "-fx-shadow-highlight-color, -fx-outer-border, " (colour :row-warning))
+                :-fx-text-fill (colour :row-warning-text)}
+
+               ".button.with-error"
```

Comment:
I wanted a 'selected' or 'on mouse down' state. it has to look like a button is being pushed